marimbabot_audio/launch/microphone.launch

```diff
@@ -0,0 +1,14 @@
+<launch>
+  <arg name="play" default="false" />
+
+  <node name="audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
+    <!-- arecord - -list-devices for `Scarlett 2i2 USB`, e.g., hw:4,0 -->
```

Comment:
Maybe add the device as an ros launch argument

Reply:
I think this line is committed, so it's doesn't matter.

Reply:
sry, I mean  commented*